ArtGalleryCRM/readme.md

```diff
@@ -0,0 +1,257 @@
+
+**Fix for Mac:**
+
+1. Close Visual Studio
+2. [Download the **PKG** installer file](https://docs.telerik.com/devtools/xamarin/installation-and-deployment/download-product-files) from your account downloads
+3. Right click on the pkg file, expand "Open with..." anbd select "Installer" (*if you see a security warning, click "Open" in the popup*)
+4. See the [Manually Add Required Assemblies](https://docs.telerik.com/devtools/xamarin/installation-and-deployment/mac/getting-started-mac#manually-add-required-assemblies) article for instructions on where to find the Binaries folder.
+5. Open the **ErpApp.sln** solution in Visual Studio again
+
+**Fix for PC:**
+
+1. Close Visual Studio
+2. [Download the **MSI** installer file](https://docs.telerik.com/devtools/xamarin/installation-and-deployment/download-product-files) from your account downloads
+3. Run the MSI installer until complete
+4. Open the **ErpApp.sln** solution in Visual Studio again
+
+
+#### Step 3. Choosing Deploy Target
+
+At this point, your Telerik assembly references are resolved and can build the solution, but we want to make sure you have the most efficient experience when building.
+
+Right-Click on the project head that you want to target and select "**Set as StartUp Project**". For example, if you want to deploy to iOS, this is what it will look like:
+
+![](.images/part3-step3-1.jpg)
+
+For the initial exploration, you'll want to choose Debug and a non-device option to keep the build time short and simple. 
+
+For iOS, choose **Debug - iPhone Simulator**:
+
+![](.images/part3-step3-2.jpg)
+
+For Android, choose **Debug - Any CPU**:
+
+![](.images/part3-step3-3.jpg)
+
+For UWP, choose **Debug - x86**:
+
+![](.images/part3-step3-4.jpg)
+
+
+#### Step 4. Update Service URL and Keys
+
+Almost done! You only need to add a few values to class library project and you can run it.
+
+1. Expand the `ArtGalleryCRM.Forms/Common` folder and open the **ServiceConstants.cs** file
+
+![](.images/part3-step4-1.jpg)
+
+2. Update the properties with your service values
+
+![](.images/part3-step4-2.jpg)
+
+* `AzureMobileAppUrl` is the Mobile App Service URL from **Part 2 - Step 3** above (e.g. `https://myappservicename.azurewebsites.net`)
+* `DirectLineSecret` is the value you got in **Part 1 - Prerequisite 2** above (e.g. `Pxw2j5nYWC8.cAw.735.XPE-L_XQg5ihVZlEK4IjMjUIWPbn4Ul1IXf5j1V7Wdp`).
+* `BotId` is the value you got in **Part 1 - Prerequisite 3** above (e.g. `MyIntelligentSupportBot`).
+
+
+#### Step 5. Build and Deploy
+
+It's showtime! Take the following steps to see the app running:
+
+1. Right-click on the start up project and select **Rebuild**
+
+![](.images/part3-step5-1.jpg)
+
+
+2. Make sure the proper target is still selected, then select **Debug > Start without Debugging** (or just use Ctrl-F5)
+
+![](.images/part3-step5-2.jpg)
+
+3. Explore the app!
+
+![](.images/part3-step5-3.jpg)
```

Comment:
Its better to show some data or active chat with the bot.

Reply:
Updated screenshot to also include bot at runtime answering 2 user questions.